components/host-dashboard/reports-section/TransactionsOverviewSection.js

```diff
@@ -270,9 +233,27 @@ const TransactionsOverviewSection = ({ host, isLoading, dateInterval }) => {
     },
   ];
 
+  /*
+   * If a date doesn't have any contributions or expenses API returns nothing.
+   * But we need to make sure the two series (expenses and contributions) show 0 in these cases rather than NaN which
+   * is shown by default by Appex charts.
+   */
+  if (series[0]?.data?.length > series[1].data?.length) {
+    series[0].data.forEach(data => {
+      if (!series[1].data.find(point => point.x === data.x)) {
+        series[1].data.push({ x: data.x, y: 0 });
```

Comment:
This is likely wrong, when I test, the values are not appearing at the right date :-(

Reply:
<img width="624" alt="Screenshot 2021-12-06 at 19 58 32" src="https://user-images.githubusercontent.com/806/144905731-148da291-da97-48b1-8d9b-5dd03a44fd45.png">